docs/API.md

```diff
@@ -276,7 +276,7 @@ Parameters.
 Each token takes the following fields for it's create method respectively:
 
 * sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType,
-                 tokenVerificationId
+                 mustVerify, tokenVerificationId
```

Comment:
Argh, and one more (last one I promise), circa line 511 in `fxa-auth-db-server/docs/API.md`